Octokit/Helpers/Pagination.cs

```diff
@@ -0,0 +1,76 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+
+namespace Octokit
+{
+    internal static class Pagination
```

Comment:
The contents of this class have fallen out of the refactoring to get pagination into the app.

I found myself duplicating this for the `Task` and `Observable` APIs, so this ended up as the simplest abstraction that could do both.

Perhaps I'll go further with the refactoring to get this feeling more natural, but for now it does the job.